client/src/components/molecules/RequestsTable.tsx

```diff
@@ -21,6 +22,15 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
         }
       }
     `;
+    const softDeleteRequest = gql`
+    mutation deleteRequest($id: ID) {
+        softDeleteRequest(id: $id) {
+            id
+            success
+            message
+        }
+    }`
+
 
     const [requests, setRequests] = useState(props.requests);
```

Comment:
Would it be possible to just initialize requests as 
```
const [requests, setRequests] = useState(props.requests.filter((request)=> request.deleted === false));
```

So then getting the number of request is just `requests.length`

Reply:
Updated.